reference/tax_provider.yml

```
@@ -923,7 +923,7 @@ components:
           $ref: '#/components/schemas/TaxClass'
         tax_exempt:
           type: boolean
-          description: 'Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. Tax-exempt items are included in the request for auditing purposes.'
+          description: 'Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. Tax-exempt items are included in the request for auditing purposes. Tax-exempt items must have a tax amount of zero within the tax quote response.'
```

```suggestion
          description: 'Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. The request body consists of tax-exempt items for auditing purposes. Tax-exempt items must have a tax amount of zero within the tax quote response.'
```
Removed passive voice

---

Thanks @bc-traciporter!

---

Ok, no problem